src/locales/en-US/common.ftl

```diff
@@ -20,3 +20,6 @@ common-experimentalTag-tooltip-title = Experimental feature
 common-error-title = An error has occurred
 common-error-message = Message
 common-error-traceID = Trace ID
+common-error-relayNetworkRequestError-anUnexpectedNetworkError = 
```

Comment:
Would these fit better in `framework.ftl`? Also, I know you are using the message that was there, but I think it should be `occurred` not `occured` in the message.

Reply:
Let me have a quick look at `framework.ftl`. I always get confused which ftl does what in this layer of the front-end.

I will also fix up the typo! 😄

Reply:
Ah, it is indeed in the framework sub-dir... into `framework.ftl` it goes!